packages/loader/container-definitions/src/loader.ts

```diff
@@ -93,6 +93,7 @@ export interface IContainerEvents extends IEvent {
     (event: "closed", listener: (error?: ICriticalContainerError) => void);
     (event: "warning", listener: (error: ContainerWarning) => void);
     (event: "op", listener: (message: ISequencedDocumentMessage) => void);
+    (event: "throttled", listener: (error: IThrottlingWarning) => void);
```

Comment:
I'm not sure we want it here.
 I think we want these kids of things to go to "warning" event, and improve it over time to provider "on" & "off" events for each type of warning. I.e. host needs to know when problem starts, and when problem ends. We need general structure that applies to all warnings, having separate events will make it harder to force same structure and similar handling on host side.

Reply:
Removed.

Reply:
I do not see it - it's still here.